src/components/page-layout/components/page-sidebar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
The only purpose of this component is to pass the `parentPage` prop to the `NavigationPage` component.